heuristics/dbic_bids.py

```diff
@@ -480,7 +484,7 @@ def get_dups_marked(info):
     #  were "cancelled"
     info = info.copy()
     dup_id = 0
-    for template in info:
+    for template, series_ids in list(info.items()):
```

Comment:
I am doing inplace extension of the dictionary, ... I guess should be ok since list here, but then next line could go away